background-concealment-demo-py/BackgroundConcealmentDemo.ipynb

```diff
@@ -0,0 +1,268 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.    g_alpha    = cv.gapi.resize(GTensorToImg.on(g_nn_out), in_size)
</pre><p>Maybe good to have a comment here, #This is a mask from an NN inference..</p><p>Or it can be just named <code>g_mask</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TolyaTalamanov/g-api-collection/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>